packages/theme/stories/introduction.stories.mdx

```diff
@@ -143,12 +134,12 @@ Consider the example given above:
 In this case, the scope is `:root`. This is equivalent to `html` so variables declared in this way are global. They can be referenced in any CSS that is applied to the page. We constrain the scope of a variable using a CSS selector, exactly as we declare regular CSS rules:
 
 ```css
-.uitk-light {
+.uitk-theme[data-mode^="light"] {
```

Comment:
Mode should always be 'light' or 'dark' so we don't need the 'starts with '(`^`) attribute selector. Unless I'm missing something